app/controllers/ReturnsController.scala

```diff
@@ -165,4 +190,16 @@ class ReturnsController @Inject()(mcc: MessagesControllerComponents,
       Redirect(routes.ReturnsController.accountPeriodsPage())
     }
   }
-}
+
+  private def getEarlierDate(date1: LocalDate, date2: LocalDate): LocalDate = {
```

Comment:
As this is only used in one place (and these controllers are already large and unruly) can this be incorporated into the below method?

Reply:
Done